build.sh

```diff
@@ -17,10 +17,10 @@ set -ex
 cd "$(dirname "${BASH_SOURCE[0]}")"
 
 #type="Release"
-# generator="Unix Makefiles"
+generator="Unix Makefiles"
 
 type=${type:-"Debug"}
-generator=${generator:-"Ninja"}
```

Comment:
Seriously? 😮‍💨

Reply:
Sorry will remove that don't know why I always forget it can be set